test-suite/src/test/java/io/micronaut/docs/server/consumes/ConsumesControllerSpec.java

```diff
@@ -77,7 +76,7 @@ void testConsumes() {
                 client.toBlocking().exchange(HttpRequest.POST("/consumes/multiple", book)
                 .contentType(MediaType.APPLICATION_JSON)));
 
-        assertThrows(CodecException.class, () ->
+        assertDoesNotThrow(() ->
```

Comment:
This was failing because toString convertion was allowed only for primitives `ClassUtils.isJavaBasicType(object.getClass())`. I might revert it.

Reply:
Reverted the change